x-pack/plugins/threat_intelligence/cypress/e2e/indicators.cy.ts

```diff
@@ -45,29 +48,68 @@ const THREAT_INTELLIGENCE = '/app/security/threat_intelligence/indicators';
 const URL_WITH_CONTRADICTORY_FILTERS =
   '/app/security/threat_intelligence/indicators?indicators=(filterQuery:(language:kuery,query:%27%27),filters:!((%27$state%27:(store:appState),meta:(alias:!n,disabled:!f,index:%27%27,key:threat.indicator.type,negate:!f,params:(query:file),type:phrase),query:(match_phrase:(threat.indicator.type:file))),(%27$state%27:(store:appState),meta:(alias:!n,disabled:!f,index:%27%27,key:threat.indicator.type,negate:!f,params:(query:url),type:phrase),query:(match_phrase:(threat.indicator.type:url)))),timeRange:(from:now/d,to:now/d))';
 
-Cypress.Cy.prototype.onUncaughtException = function () {};
+describe('Invalid Indicators', () => {
+  describe('verify the grid loads even with missing fields', () => {
+    before(() => {
+      esArchiverLoad('threat_intelligence/invalid_indicators_data');
 
-describe('Indicators', () => {
-  before(() => {
-    esArchiverLoad('threat_intelligence/invalid_indicators_data');
-  });
-  after(() => {
-    esArchiverUnload('threat_intelligence/invalid_indicators_data');
+      cy.visit(THREAT_INTELLIGENCE);
+      selectRange();
+    });
+    after(() => {
+      esArchiverUnload('threat_intelligence/invalid_indicators_data');
+    });
+
+    it('should display data grid despite the missing fields', () => {
+      cy.get(INDICATORS_TABLE).should('exist');
+
+      // there are 19 documents in the x-pack/test/threat_intelligence_cypress/es_archives/threat_intelligence/invalid_indicators_data/data.json
+      const documentsNumber = 19;
+      cy.get(INDICATORS_TABLE_ROW_CELL).should('have.length.gte', documentsNumber);
+
+      // the last document has no hash so the investigate in timeline button isn't rendered
+      cy.get(INDICATORS_TABLE_INVESTIGATE_IN_TIMELINE_BUTTON_ICON).should(
+        'have.length',
+        documentsNumber - 1
+      );
+
+      // we should have 19 documents plus the header
+      cy.get(INDICATORS_TABLE_INDICATOR_NAME_CELL).should('have.length', documentsNumber + 1);
+
+      // the last entry has no hash to we show - in the Indicator Name column
+      cy.get(INDICATORS_TABLE_INDICATOR_NAME_CELL).last().should('contain.text', '-');
+    });
   });
 
-  describe('verify the grid loads even with missing fields', () => {
+  describe('verify the grid loads even with missing mappings and missing fields', () => {
```

Comment:
same tests as above but this time with a missing `mappings.json` file. Not sure these tests are really useful though...